students/eunhyung/users/views.py

```diff
@@ -37,13 +37,26 @@ def post(self, request):
             )
             return JsonResponse({'message':'SUCCESS'}, status=201)
             
-        except:
+        except KeyError:
             return JsonResponse({"message": "KEY_ERROR"}, status=400)
     
-        
-    
-        
-       
-        
-        
-        
+
+class SignInView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            email    = data['email']
+            password = data['password']
+            
+            email_exist_user = User.objects.get(email=email)
+                        
+            if password == email_exist_user.password:
+                return JsonResponse({'message':'SUCCESS'}, status=200)
+            else:
+                return JsonResponse({'message':'INVALID_USER'}, status=401)
```

Comment:
일치하지 않는 경우부터 생각한다면 else을 사용하지 않고 조금 더 가독성을 높일 수 있을 것 같습니다.
```suggestion
            if password != email_exist_user.password:
                return JsonResponse({'message':'INVALID_USER'}, status=401)

            return JsonResponse({'message':'SUCCESS'}, status=200)
```

Reply:
앗 넵!! try문 내부에서 로그인에 실패하는 경우를 먼저 if문으로 걸러내고 나서 로그인을 처리하도록 수정 사항 반영했습니다!